css/style.css

```diff
@@ -107,6 +107,18 @@ img:hover {
 	cursor: pointer;
 }
 
+#fullscreen-button {
+	transition: all 0.2s;
+	margin: 5px;
+	height: 60px;
```

Comment:
height is an attribute of image, I am not sure if this will have any effect on the button